wagtail/wagtailadmin/templates/wagtailadmin/pages/add_subpage.html

```diff
@@ -13,15 +13,15 @@
 
         {% if page_types %}
             <ul class="listing">
-                {% for verbose_name, app_label, model_name in page_types %}
+                {% for verbose_name, page_help_text, app_label, model_name in page_types %}
                     <li>
                         <div class="row row-flush">
                             <div class="col6">
                                 <a href="{% url 'wagtailadmin_pages:add' app_label model_name parent_page.id %}{% if next %}?next={{ next }}{% endif %}" class="icon icon-plus-inverse icon-larger">{{ verbose_name }}</a>
                             </div>
 
                             <small class="col6" style="text-align:right">
-                                <a href="{% url 'wagtailadmin_pages:type_use' app_label model_name %}" class="nolink">{% blocktrans with page_type=verbose_name %}Pages using {{ page_type }}{% endblocktrans %}</a>
```

Comment:
I think, we still need this "Pages using Blog Page" text. If we replace this with page's help text ("Use this page to create a blog post", for example), the purpose of the link will be unclear.

I think, we should display the help text separately. Underneath the page's verbose name.

![screen shot 2017-10-02 at 10 33 04](https://user-images.githubusercontent.com/509198/31071938-ded86478-a75d-11e7-8707-4d37d258153a.png)